Editor/Scripts/CodeGenerators/RGActionCodeGenerator.cs

```diff
@@ -354,61 +414,85 @@ private static string GetFullTypeNameWithNamespace(ITypeSymbol typeSymbol, Seman
             return namespacePrefix + typeSymbol.Name;
         }
 
+
+        private static string GetActionKey(RGActionInfo action, bool includeAssembly)
```

Comment:
What is meant by key here? I see this is returning the namespace, should this actually be GetActionNamespace? A comment on both of these functions would be helpful!

Reply:
@vontell `Type.GetType` requires the fully qualified type of the component and the assembly it's in, as a string with the format: "{namespace}.{typeName}, {assemblyName}" so it's a bit more than the namespace. It's the lookup key for GetType.

Comment would probably be helpful for that :)

Reply:
I liked your explanation so heavily borrowed from it ;-)
@vontell let me know if this makes sense now

Reply:
Makes sense to me!